build/build.xml

```diff
@@ -48,6 +49,12 @@ External Dependencies:
 	<echo>Loading base properties</echo>
 	<loadproperties srcFile="build.properties"/>
 
+	<!-- Load secret properties -->

```

Comment:
What is the purpose of these settings? Looks like we need a properties file in the /root. What is needed there?